ACS event updates

JoshLove-msft opened the pull request:
Fixes https://github.com/Azure/azure-sdk-for-net/issues/19603

ellismg commented on sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/SystemEventNames.cs:
```diff
@@ -32,7 +32,7 @@ public static class SystemEventNames
         /// <see cref="AcsChatParticipantAddedToThreadEventData"/> system event.
         /// </summary>
-        public const string AcsChatParticipantAddedToThread = "Microsoft.Communication.ChatParticipantAddedToThread";
+        public const string AcsChatParticipantAddedToThread = "Microsoft.Communication.ChatThreadParticipantAdded";
```
Feels like we need to add something for `Microsoft.Communication.RecordingFileStatusUpdated` to this file?

JoshLove-msft replied:
Yep, looks like the test actually works! https://dev.azure.com/azure-sdk/public/_build/results?buildId=796869&view=ms.vss-test-web.build-test-results-tab&runId=17558288&resultId=100009&paneView=debug

ellismg reviewed (changes requested):
I think you might need to add something to SystemEventNames.cs for the new event?

ellismg reviewed (approved):
LGTM - It might be nice to put a little more detail in the commit message about what the problem was and how we decided to fix it (not changing the names of the properties but changing their values, because we didn't want to introduce a break), which may be helpful for folks down the line who notice the inconsistency, run and run `git blame` to try to figure out why things are the way they are.

Very cool that you had that test that failed when we forgot to add the event to the mapping table, great job on that.

JoshLove-msft committed:
Add new ACS events and update the constant values for
AcsChatParticipantAddedToThread and AcsChatParticipantRemovedFromThread
as the values were changed in the swagger.
Note that the constant names are not being changed to avoid build breaks.

JoshLove-msft commented:
> LGTM - It might be nice to put a little more detail in the commit message about what the problem was and how we decided to fix it (not changing the names of the properties but changing their values, because we didn't want to introduce a break), which may be helpful for folks down the line who notice the inconsistency, run and run `git blame` to try to figure out why things are the way they are.
> 
> Very cool that you had that test that failed when we forgot to add the event to the mapping table, great job on that.

Sure updated - for things like this I usually end up needing to look at the PR that made the change, but any additional breadcrumbs we can add should help as well.

JoshLove-msft commented:
/azp run net - eventgrid - tests

JoshLove-msft commented:
/azp run net - eventgrid - tests
